docs/Users/Tutorials/Tutorial.rst

```diff
@@ -187,25 +188,14 @@ That looks horrible, doesn't it? Let's fix it!
 ::
 
     $ coala -S python.bears=PEP8Bear python.files=\*\*/\*.py \
-    python.default_actions=PEP8Bear:ApplyPatchAction --save
+    --apply-patches --save
     # other output ...
     Executing section python...
     [INFO][11:03:37] Applied 'ApplyPatchAction' for 'PEP8Bear'.
     [INFO][11:03:37] Applied 'ApplyPatchAction' for 'PEP8Bear'.
 
 *coala* would now fix all spacing issues and without bothering you again.
 
-Currently following actions are available:
```

Comment:
- patches are shown by default now if they are small enough
- users usually only want to apply patches

This kind of documentation may go somewhere else but not in the tutorial IMO